web-client/src/views/TrialSessionDetail/TrialSessionDetail.jsx

```diff
@@ -59,6 +60,30 @@ export const TrialSessionDetail = connect(
                 tabName="EligibleCases"
                 title="Eligible Cases"
               >
+                {formattedTrialSessionDetails.isHybridSession && (
+                  <div className="grid-row grid-col-9">
+                    <div className="grid-col-1 padding-top-05 margin-right-3">
+                      <h3 id="filterHybridSessionHeading">Filter by</h3>
```

Comment:
🔧 To make the filters more accessible, I believe we are now making the text `<label>` elements so they can be read by screenreaders. We did something similar for another story in `web-client/src/views/DocketRecord/DocketRecordHeader.jsx`, line 71.

Reply:
Thanks for pointing this out.  I had this as a label at one point for accessibility.  I must have accidentally reverted it back at some point.